front/app/components/PageForm/index.tsx

```diff
@@ -81,12 +61,7 @@ const PageForm = ({
           formatMessage(messages.titleMissingOneLanguageError)
         ),
       }),
-    ...(!hideSlugInput && {
```

Comment:
Slug can not be specified anymore for pages that can be customized through PageForm, so we can delete it.